packages/prompts/src/index.ts

```diff
@@ -624,3 +624,41 @@ Only return Node.js code. DO NOT include any English text before or after the No
 
 `;
 };
+
+export const englishStyleGuidePrompt = `# English Style Guide
+
+## Instructions
+
+Your goal is to apply two style guides to any English you find in the code:
+
+1. Google's Developer docs style guide (https://developers.google.com/style)
+2. Dreyer's English
+
+You might see English in places like the following (this is not an exhaustive list):
+
+- Code comments
+- Code strings
+- Any string you find in the code (e.g. the values of properties, variables, etc.)
+- Markdown (headings, text, lists, etc.)
+- HTML attributes, like img alt tags
+- etc.
+
+Please apply the rules of the style guide to ANY English you find.
+
+## Input format
+
+Below, I'm passing data as a CSV of the following format:
+
+- \`key\`: A string, a composite key of the following format: \`<file path>:<line number>\`
+- \`code\`: A string, the code found at the \`key\` location
+
+## Output format
+
+You MUST output your results as a CSV of the following format:
+
+- \`key\`: A string, a composite key of the following format: \`<file path>:<line number>\` found in the \`key\` field of the input CSV text provided.
+- \`corrected_code_or_text\`: A string, the corrected found at the \`key\` location, with all English corrected according to the rules of the style guide.
+- \`reason\`: A string, the rule you applied to the English found at the \`key\` location. Explain why you made the change you did. If multiple rules / issues are found, list them all.
```

Comment:
```suggestion
- `reason`: A string, the rule(s) you applied to the English found at the `key` location. Explain why you made the change(s). If multiple rules or issues are found, list them all.
```

Reply:
Changed "rule" to "rule(s)" and "change" to "change(s)" to account for multiple rules or changes.